src/plugins/discover/public/application/main/components/layout/discover_main_content.tsx

```diff
@@ -72,6 +72,8 @@ export const DiscoverMainContent = ({
       if (trackUiMetric) {
         if (mode === VIEW_MODE.AGGREGATED_LEVEL) {
           trackUiMetric(METRIC_TYPE.CLICK, FIELD_STATISTICS_VIEW_CLICK);
+        } else if (mode === VIEW_MODE.PATTERN_LEVEL) {
+          trackUiMetric(METRIC_TYPE.CLICK, PATTERN_ANALYSIS_LOADED);
```

Comment:
This should be PATTERN_ANALYSIS_VIEW_CLICK

Reply:
Updated in https://github.com/elastic/kibana/pull/178916/commits/186f78b53ebc6cf2166bb73a2daffa6f267ef003